lib/api/index.js

```diff
@@ -132,6 +132,10 @@ class ApiLoader {
   static makeAssertProxy(api) {
     return new Proxy(api, {
       get(target, name) {
+        if (name === '__isProxy') {
```

Comment:
why do we need this and where are we setting `__isProxy` ?

Reply:
This is required to identify whether a property in `namespacedApi` is set to a Proxy or not (more specifically `assert` and `verify`). The problem is that there is no inbuilt way in JS to know if a value is a Proxy or not, like we cannot just do `namespacedApi.assert instanceof Proxy`. So, adding the above line in proxy allows us to do `namespacedApi.assert.__isProxy` and it will return `true`.

We are not setting `__isProxy` anywhere, but just checking it in `core/client.js` while clearing the namespacedApi for a new session.

Reply:
got it